### PR TITLE
chore: Fix Dependabot config and adjust groups

### PR DESCRIPTION
**Pull request by Dinnerbone**
Removes renovate, too.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -25,14 +32,19 @@ updates:
         - "wgpu"
         - "naga"
         - "naga_oil"
-        - "egui-winit"
-        - "egui-wgpu"
+        - "egui*"
```
Comment (torokati44) [inline .github/dependabot.yml:35]: The group name could probably be tweaked now, but just a nit...

**Comment by torokati44** [comment]
For reference: https://github.com/dependabot/dependabot-core/issues/2936#issuecomment-2037599286

File: .github/dependabot.yml [hunk 2]
```diff
@@ -25,14 +32,19 @@ updates:
...
         - "web-sys"
         - "wasm-bindgen-futures"
+      gradle-minor:
+        patterns:
+          - "*"
+        update-types:
+          - "minor"
+          - "patch"
     ignore:
       - dependency-name: "tracing-tracy"
```
[the remark this answers is not in the document]
Comment (Dinnerbone) [reply .github/dependabot.yml]: Added

**Review by torokati44 — APPROVED** [review]
_Now lessee!_

File: .github/dependabot.yml [hunk 3]
```diff
@@ -1,38 +1,51 @@
...
         patterns:
         - "winit"
         - "wgpu"
         - "naga"
         - "naga_oil"
-        - "egui-winit"
-        - "egui-wgpu"
+        - "egui*"
         - "raw-window-handle"
       wasm-bindgen-dependencies:
```
Comment (torokati44) [inline .github/dependabot.yml:37]: I guess the `-dependencies` part is also really redundant here, but oh well...
Can be fixed next time someone touches it.

Comment (torokati44) [reply .github/dependabot.yml:37]: Turns out, that someone is me... :clown_face: 
https://github.com/ruffle-rs/ruffle/pull/15873

**Pull request merged**